lib/fix-google-html.js

```diff
@@ -122,6 +122,23 @@ export function fixNestedLists (node) {
   });
 }
 
+// this seems to only occur when running via nodejs (not the browser): there's a top-level <b> tag
+// this is only called once with the parent node
+function removeRootBoldWrapper(node) {
+  // there are some cases, like translating <b>something something</b> where we don't want to remove the root node
+  if(node.children.length === 1 && node.children[0].tagName === 'b') {
+    return
+  }
+
+  for(let i = 0; i < node.children.length; i++) {
+    const child = node.children[i];
+
+    if(child.tagName === 'b') {
+      node.children.splice(i, 1, ...child.children);
+    }
+  }
+}
```

Comment:
Can you explain more about what the purpose here is? Why are you concerned particularly with `<b>` elements here, and what circumstances are you encountering? If this is a CLI-specific issue, it should *probably* be handled in the CLI portion of the code, not here.

Reply:
Added more in the comments! Let me know if this still isn't clear.